src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        super.draw();
+        System.out.println("Figure: circle, radius: " + radius + " color: " + Color.WHITE);

```

Comment:
Use 'name()' for getting String representation of enum constants instead of 'toString()' or other methods.